fml/thread.cc

```diff
@@ -67,6 +73,14 @@ void Thread::SetCurrentThreadName(const Thread::ThreadConfig& config) {
   SetThreadName(config.name);
 }
 
+std::string Thread::GetCurrentName() {
+  return *(tls_thread_name.get());
+}
+
+ThreadId Thread::GetCurrentId() {
+  return std::this_thread::get_id();
```

Comment:
Here maybe we should call something like `syscall(__NR_gettid)` or `pthread_threadid_np` to get the system-wide unique integral ID of the thread.